tests/tests_integration/test_api/test_hosted_extractors/test_jobs.py

```diff
@@ -0,0 +1,135 @@
+from __future__ import annotations
+
+import pytest
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes.hosted_extractors import (
+    CogniteFormat,
+    Destination,
+    Job,
+    JobList,
+    JobLogsList,
+    JobMetricsList,
+    JobUpdate,
+    JobWrite,
+    Source,
+)
+from cognite.client.exceptions import CogniteAPIError
+from cognite.client.utils._text import random_string
+
+# @pytest.fixture
+# def one_source(cognite_client: CogniteClient) -> Source:
+#     my_source = EventHubSourceWrite(
+#         external_id=f"myNewSource-{random_string(10)}",
+#         host="myhost",
+#         event_hub_name="myeventhub",
+#         key_name="mykeyname",
+#         key_value="secret",
+#     )
+#     retrieved = cognite_client.hosted_extractors.sources.retrieve(my_source.external_id, ignore_unknown_ids=True)
+#     if retrieved:
+#         yield retrieved
+#     created = cognite_client.hosted_extractors.sources.create(my_source)
+#     yield created
+#
+#     cognite_client.hosted_extractors.sources.delete(created.external_id, ignore_unknown_ids=True)
```

Comment:
Was this uncommented because `Source` was not merged yet? It seems to be merged now so maybe uncomment or remove?